server/service/individualServices/regardapplicationcallback.js

```diff
@@ -11,16 +11,19 @@ const FcPort = require('onf-core-model-ap/applicationPattern/onfModel/models/FcP
 const forwardingConstructAutomationInput = require('onf-core-model-ap/applicationPattern/onfModel/services/models/forwardingConstruct/AutomationInput');
 const httpClientInterface = require('onf-core-model-ap/applicationPattern/onfModel/models/layerProtocols/HttpClientInterface');
 const logicalTerminationPoint = require('onf-core-model-ap/applicationPattern/onfModel/models/LogicalTerminationPoint');
+const stringProfile= require('onf-core-model-ap/applicationPattern/onfModel/models/profile/StringProfile')
 
 exports.CreateLinkForInquiringBasicAuthApprovals = async function (applicationName, releaseNumber, reqheaders) {
     return new Promise(async function (resolve, reject) {
         try {
             let InquiringOamRequestCreateLinkForwardingName = "RegardApplicationCausesSequenceForInquiringBasicAuthRequestApprovals.CreateLinkForInquiringBasicAuthApprovals";
             let InquiringOamRequestCreateLinkRequestBody = {};
-
+            let stringProfileList = await stringProfile.getStringProfile('aa-2-1-2-string-p-001')
```

Comment:
Instead of getting the string profile from the uuid , get it from the string-name NameOfOperationForInquiringApprovals